README.md

```diff
@@ -42,6 +42,18 @@ Some crates of the **matrix-rust-sdk** can be embedded inside other
 environments, like Swift, Kotlin, JavaScript, Node.js etc. Please,
 explore the [`bindings/`](./bindings/) directory to learn more.
 
+## Testing
+
+You can run most of the tests that also happen in CI also using `cargo xtask ci`.
+This needs a few dependencies to be installed, as it also runs automatic WASM tests.
+If you want to execute only one part of CI, there is a sub-command for it.
+
+Some tests are not automatically run in `cargo xtask ci`, for example the integration tests,
+that need a running synapse instance. These tests reside in `./testing/matrix-sdk-integration-testing`.
+See its [README](./testing/matrix-sdk-integration-testing/README.md) to easily set up a synapse for testing purposes.
+
+All system-compatible tests can then be run with `cargo nextest run --workspace` or `cargo test --workspace`.
```

Comment:
Since this uses default features it will miss some things. I think we shouldn't encourage this, people should go through xtask.

Reply:
Then we need integration tests in xtask as well :D

Reply:
I was just talking about the last paragraph, not the one before about integration tests.

Reply:
Yeah removed it and added a notice to run the integration tests in the integration test README instead.